<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageServerDefsDialog</class>
 <widget class="QDialog" name="ManageServerDefsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Server Definitions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtclient/qtclient.qrc">
    <normaloff>:/images/world-setup.png</normaloff>:/images/world-setup.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="vLayoutLeft">
     <item>
      <widget class="QLabel" name="definitionLbl">
       <property name="text">
        <string>Configured Servers:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="definitionList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutRight">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="showParamLbl">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newBttn">
       <property name="text">
        <string>&amp;New Definition ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editBttn">
       <property name="text">
        <string>&amp;Edit Definition ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBttn">
       <property name="text">
        <string>&amp;Delete Definition</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultBttn">
       <property name="text">
        <string>&amp;Set as default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>newBttn</tabstop>
  <tabstop>editBttn</tabstop>
  <tabstop>deleteBttn</tabstop>
 </tabstops>
 <resources>
  <include location="../qtclient/qtclient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManageServerDefsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManageServerDefsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
